### PR TITLE
Jessica Sandler

### PR DESCRIPTION
**Pull request by jessand77**
https://jessicas-news-site-project.netlify.app/

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,213 @@
...
+  margin: 0;
+  background-color: #9DC88D;
+}
+
+h1, h2 {
+  font-family: 'Neuton', Garamond, serif;
+}
+
+p {
+  font-family: 'Work sans', Helvetica, sans-serif;
```
Comment (EmmaaBerg) [inline code/style.css:20]: The same font-family is added/styled for the HTML in row 6 so I think this one can be removed.

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,213 @@
...
+  line-height: 1.4;
+}
+
+a {
+  font-family: 'Work sans', Helvetica, sans-serif;
```
Comment (EmmaaBerg) [inline code/style.css:25]: The same font-family is added/styled for the HTML in row 6 so I think this one can be removed as well.

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,213 @@
...
+  margin: 0 auto;
+  /*Centers the main container*/
+}
+
+/* Big news section */
+
+.big-news-section {
+  display: grid;
+  grid-template-rows: 1fr auto;
+  grid-template-columns: 1fr;
```
Comment (EmmaaBerg) [inline code/style.css:80]: Is it possible to get the same result by only using column for the mobile version? In my mind I think this just gives one cell if there is one column and one row,

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,213 @@
...
+  }
+
+  .header {
+    justify-content: space-between;
+  }
+
+  /*Hides the header h1*/
+  .header h1 {
+    display: none;
+  }
```
Comment (EmmaaBerg) [inline code/style.css:169]: I would like to see the name of the page for all different screen sizes. That was something I missed when looking at the page for the tablet size.

File: code/index.html [hunk 5]
```diff
@@ -1,22 +1,154 @@
...
+    <header class="header">
+      <div class="logo-container"><img src="./images/carrot-icon.png" alt="carrot-icon" /></div>
+      <h1>Veggie News</h1>
+      <nav class="header-nav">
+        <ul>
+          <li><a href="#">About us</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="#">Nutrition</a></li>
+        </ul>
+      </nav>
```
Comment (EmmaaBerg) [inline code/index.html:33]: Good that the nav only contain links.

**Review by EmmaaBerg — COMMENTED** [review]
Overall I thought the code had a good structure and was easy to follow. It follows the blue as well as the general requirements. I added som few comments mainly regarding the CSS.

File: code/style.css (hunk 2 above)
Comment (jessand77) [reply code/style.css:25]: You are right, I'll remove the font-family.

File: code/style.css (hunk 1 above)
Comment (jessand77) [reply code/style.css:20]: You are right, I'll remove the font-family.

File: code/style.css (hunk 3 above)
Comment (jessand77) [reply code/style.css:80]: I'm not sure but I'll try to find out if it's possible.

File: code/style.css (hunk 4 above)
Comment (jessand77) [reply code/style.css:169]: Yes, I was thinking about that too... I'll change it.

**Pull request closed without merging**